recipes/python-3.5/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    export CFLAGS="-I$PREFIX/include $CFLAGS"
+    export LDFLAGS="-L$PREFIX/lib -headerpad_max_install_names $LDFLAGS"
+    sed -i -e "s/@OSX_ARCH@/$ARCH/g" Lib/distutils/unixccompiler.py
+    ./configure --enable-shared --enable-ipv6 --with-ensurepip=no \
+        --prefix=$PREFIX
+fi
+if [ `uname` == Linux ]; then
+    ./configure --enable-shared --enable-ipv6 --with-ensurepip=no \
+        --prefix=$PREFIX \
+        --with-tcltk-includes="-I$PREFIX/include" \
+        --with-tcltk-libs="-L$PREFIX/lib -ltcl8.5 -ltk8.5" \
+        CPPFLAGS="-I$PREFIX/include" \
+        LDFLAGS="-L$PREFIX/lib -Wl,-rpath=$PREFIX/lib,--no-as-needed"
+fi
+
+make
+make install
+ln -s $PREFIX/bin/python3.5 $PREFIX/bin/python
+ln -s $PREFIX/bin/pydoc3.5 $PREFIX/bin/pydoc
+
+if [ `uname` == Darwin ]; then
+    # Some extension modules are renamed when importing fails do to a missing
```

Comment:
The libraries and available and they are found by the compiler but not by the loader when the importability of the [modules is tested by the build](https://travis-ci.org/conda-forge/staged-recipes/jobs/124085198#L1553) because `install_name_tool` has not been run to fix the library paths.  This triggers the renaming of the module.  There might be a way properly set the rpath during the compile but I haven't figured it out.